### DISCUSSION TITLE
Design Idea: Cave Exploration

### DISCUSSION THREAD

**Discussion by mokun** [mars-sim/mars-sim#449] [Ideas]
![lavatube](https://user-images.githubusercontent.com/1168584/65381111-f54e3e80-dc9e-11e9-872b-b6aa4c1191ec.jpg)

There have been numerous satellite pictures of (collapsed) interesting looking *lava tubes* circulating on the internet near the southwest area of the Pavonis Mons

Would it be a good place for mars-simmer to start a settlement in that area ?

What keys design concept do we need to consider in revising our core engine in recognition of the existence of these cave/lava tubes ? 

This [wiki](https://en.wikipedia.org/wiki/Martian_lava_tube) explains the internal mechanic of the lava tubes.

THEMIS has probably the most recent [picture](http://redplanet.asu.edu/?p=26368) on it. 

As we are looking forward to planning for the eventual first landing of human beings on Mars, it's getting more real that we might just select the landing site close to this region.

See possible landing sites at https://www.usgs.gov/media/images/location-caves-tharsis

**Comment by mokun** [comment]
See [this](https://astrogeology.usgs.gov/search/map/Mars/MarsReconnaissanceOrbiter/CTX/HumanExplorationZones/CTX_Human_EZ_sites_Oct2015) for more candidate landing sites.

**Reply by mokun** [reply]
To boost our imagination further, see this [video](https://www.youtube.com/watch?v=tCcx93NIPWM) regarding the future robotic and human and exploration of the lava tubes on Moon and Mars.

We can in fact create a Lava Tube Exploration mission here in mars-sim.

**Comment by mokun** [comment]
Read this interesting, comprehensive 100-page International Space University [paper](https://www.academia.edu/4608351/ACCESS_Mars_Final_Report_Revised_3) from 2009.

**Comment by mokun** [comment]
@bevans2000 @Urwumpe 

Do you see the prospect of having a Lava tube field/exploration mission in near future ?

Building a way-station in the lava tube for collecting scientific, weather, resource data would be valuable.

It may take the whole settlement to devote themselves into doing this.

**Reply by bevans2000** [reply]
I see it as possible. But first we probably need a Settlement to have the ability to build another Settlement (way station). I hope the massive changes I am doing to construction will make it run better without the problems
